ivadomed/loader/mri2d_segmentation_dataset.py

```diff
@@ -178,7 +178,7 @@ def __getitem__(self, index):
             seg_pair_slice, roi_pair_slice = copy.deepcopy(self.indexes[index])
 
         # In case multiple raters
-        if seg_pair_slice['gt'] is not None and isinstance(seg_pair_slice['gt'][0], list):
+        if seg_pair_slice['gt'] and isinstance(seg_pair_slice['gt'][0], list):
```

Comment:
Wow. Such small line yet has big impact. A "Truthy" check vs "is None" / is empty array check is really the core of this problem. Thanks for spotting this effectively.

Reply:
Also, something else I noticed not from your PR but from this class in general is this habitual direct referencing dict keys for retrieval. I think it is an [possible antipattern](https://deepsource.io/blog/8-new-python-antipatterns/#5-not-using-get-to-return-default-values-from-a-dictionary) that we should address in one of the dev meetings.